gnucash/gnome-utils/dialog-transfer.c

```diff
@@ -275,18 +275,35 @@ lookup_price(PriceReq *pr, PriceDate pd)
     if (gnc_commodity_equiv(gnc_price_get_currency(prc), pr->from))
```

Comment:
This whole if clause is only to print a debug message. The added logic doesn't make sense. If prc doesn't exist no price is found. Period. The concept of reverse or normal doesn't apply in that case. Only when prc exists you can test for reverse or not. So the test sequence should be:
1. is prc NULL ? => info message no price found and return FALSE
2. is reverse ? => info message reporting the found reverse conversion rate
3. otherwise => info message reporting the normal conversion rate
In the end set pr->price and return TRUE

Reply:
Brain dead, just needed to move prc not null test to before the info stuff.